scripts/migration/migrate-unix.sh

```diff
@@ -1,66 +1,70 @@
-#!/bin/sh
+#!/bin/bash
 
-echo -e "********************************************"
-echo -e "      Ferdium User Data Migration Tool      "
-echo -e "          WARNING: UNIX OS Only!            "
-echo -e "********************************************"
-echo -e "\n\nThis tool migrates your user data from Ferdi to Ferdium."
-echo -e "Intended to be used on Linux or MacOS machine.\n\n"
+set -e
+
+echo "********************************************"
+echo "      Ferdium User Data Migration Tool      "
+echo "          WARNING: UNIX OS Only!            "
+echo "********************************************"
+echo "\n\nThis tool migrates your user data from Ferdi to Ferdium."
+echo "Intended to be used on Linux or MacOS machine.\n\n"
 read -p "Do you want to proceed? (y/N) " confirm
 
 case "$confirm" in
-    y|Y|[yY][eE][sS] ) echo -e "Starting...";;
-    n|N|[nN][oO] ) exit 1;;
-    * ) exit 1;;
+  y|Y|[yY][eE][sS] ) echo "Starting...";;
+  n|N|[nN][oO] ) exit 1;;
+  * ) exit 1;;
 esac
 
 os_check=$(uname)
-echo "Your OS is $os_check"
 
-if [ $os_check == "Linux" ];then
-  FERDI_PATH="$HOME/.config/Ferdi"
-  FERDIUM_PATH="$HOME/.config/Ferdium"
-elif [ $os_check == "Darwin" ];then
-  FERDI_PATH="$HOME/Library/Application Support/Ferdi"
-  FERDIUM_PATH="$HOME/Library/Application Support/Ferdium"
+if [ $os_check == "Linux" ]; then
+  echo "Your OS is: Linux"
+  BASE_PATH="$HOME/.config"
+elif [ $os_check == "Darwin" ]; then
+  echo "Your OS is: MacOS"
+  BASE_PATH="$HOME/Library/Application Support"
 else
   echo "Your OS is not supported by this script"
   exit 1
 fi
 
+FERDI_PATH="$BASE_PATH/Ferdi"
+FERDIUM_PATH="$BASE_PATH/Ferdium2"
```

Comment:
I think there is a typo on the script after refactoring, it should be Ferdium instead of Ferdium 2

Reply:
thanks for catching that @Evan-aja - that was for my testing. I have added a new commit to fix it now!